docs/production-deployment/self-hosted-guide/checklist.mdx

```diff
@@ -23,9 +23,39 @@ tags:
   - versioning
 ---
 
-Consider the following when running a production level Temporal Service.
+This page is intended to describe the most common challenges for customers who choose to self-host Temporal and share recommendations to mitigate those challenges. 
 
-## Scaling and Metrics {#scaling-and-metrics}
+Temporal at its core is about durability and reliability.
+This durability is compromised if it is not architected and deployed as a reliable, trusted platform.
+This guide provides a path to demonstrate that Temporal consumers can be confident in a Temporal service and provides a list of key tests you as a user should perform against the service, such as:
+
+This guide provides a path to demonstrate that Temporal consumers can be confident in a Temporal service and provides a list of key tests you as a user should perform against the service.
+
+## Self-Hosting Challenge Areas
+
+Significant engineering must be done and ongoing effort is required to resolve these challenges:
+- Scalability with spiky or growing workloads
+- Global hosting
+- Uptime, availability and reliability
+- Management and control plane
+- Latency, which must be kept low and consistent
+- [Security](/self-hosted-guide/security)
+- Maintenance and upgrades
+- Expert support to users of the service
+- Cost management
+
+It is critical to demonstrate confidence in these areas as part of building a mission critical Temporal Service.
+Without demonstrated architectural durability, the value of Temporal's [Durable Execution](https://temporal.io/how-it-works) model is compromised.
+
+## Scalability with Variable or Growing Workloads {#scaling-and-metrics}
+
+Workloads can be highly variable, and sustained workload spikes of 5-10x average is common. 
```

Comment:
Why are Workloads so variable. This again feels rushed, without introducing the fact that people use Workloads under various circumstances -- from Black Friday to a slow warm Sunday (not that I'm recommending those words)

Reply:
Good point -- removed some of the excessively detailed-yet-vague numbers here